test/utils/FSHLogger.test.ts

```diff
@@ -0,0 +1,50 @@
+import { logger, stats } from '../../src/utils/FSHLogger';
+
+describe('FSHLogger', () => {
+  beforeEach(() => {
+    stats.reset();
+    expect(stats.numInfo).toBe(0);
+    expect(stats.numWarn).toBe(0);
+    expect(stats.numError).toBe(0);
```

Comment:
Super minor, but I'd avoid putting assertions into the `beforeEach`.  Instead, have `beforeEach` do what it needs to do (`stats.reset()`) and create a new test that validates that `reset()` works correctly.